scripts/update_instance_data/pricing-updater/Makefile

```diff
@@ -0,0 +1,57 @@
+NAME=$(shell basename $(TOP_DIR))
+
+GIT_COMMIT=$(shell git rev-parse --short HEAD)
```

Comment:
Probably does not matter too much here, but throughout kip, we use `git describe --dirty` to capture the build version.